include/seqan3/alphabet/quality/all.hpp

```diff
@@ -65,8 +68,10 @@
  * For other formats, like Solexa and Illumina 1.0 to 1.7 the type
  * seqan3::phred68legacy is provided. To cover also the Solexa format, the Phred
  * score is stored as a <B>signed</B> integer starting at -5.
+ * If you want to store PacBio HiFi reads, we recommend to use seqan3::phred93, as these reads are stored in SAM/BAM
+ * format and thus use the full range of the phred quality scores.
```

Comment:
```suggestion
 * If you want to store PacBio HiFi reads, we recommend to use seqan3::phred93, as these
 * use the full range of the phred quality scores.
```

Illumina is also stored in SAM/BAM so this might be confusing :)